lib/coding-browser.js

```diff
@@ -61,13 +58,13 @@ async function readMultihash (reader) {
   return multihash
 }
 
-async function readCid (reader) {
+async function readCid (multiformats, reader) {
   const first = await reader.exactly(2)
   if (first[0] === 0x12 && first[1] === 0x20) {
     // cidv0 32-byte sha2-256
     const bytes = await reader.exactly(34)
     reader.seek(34)
-    return new CID(bytes)
+    return new multiformats.CID(0, 0x70, Uint8Array.from(bytes))
```

Comment:
I'd prefer constants over magic numbers. I can also understand the ones of the of if statement, but only because I'm so deep into the multiformats stuff.

Reply:
I made them all constants:

```js
const CIDV0_BYTES = {
  SHA2_256: 0x12,
  LENGTH: 0x20,
  DAG_PB: 0x70
}
```